src/components/ItemEditorPage/LeftPanel/Items/Items.tsx

```diff
@@ -42,8 +42,12 @@ export default class Items extends React.PureComponent<Props, State> {
   }
 
   componentDidMount() {
-    const { items, onSetReviewedItems } = this.props
+    const { items, selectedItemId, onSetItems, onSetReviewedItems } = this.props
     onSetReviewedItems(items)
+    const initialActiveItem = items.find(item => item.id === selectedItemId)
+    if (initialActiveItem) {
+      onSetItems([initialActiveItem])
```

Comment:
Quick question: Instead of doing this in the DidMount, can we do it in the sagas that creates the Item? If not, here is fine :)

Reply:
@juanmahidalgo -- This step is to show items when redirected using the `Preview Item in Editor` action

Reply:
oh right, sorry! Can we assign it as the selectedItemId before redirecting to the editor then?

Reply:
sure! let me test that way